tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+	"compilerOptions": {
+		"noImplicitAny": true,
+		"outDir": "build/cross-echo",
+		"target": "es2018",
+		"module": "commonjs",
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected object keys to be in natural insensitive ascending order. 'module' should be before 'target'.](https://app.codacy.com/gh/cpuabuse/cross-echo/pullRequest?prid=5213902)